src/build/Wpf/Wpf.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project>
```

Comment:
@onovotny ❓ How do I get this included as one of the .props files?

Reply:
I would suggest a wildcard import from https://github.com/onovotny/MSBuildSdkExtras/blob/master/src/build/netstandard1.0/MSBuild.Sdk.Extras.props that matches the one in the Common.targets file but looking for *.props. Then those props can use conditional groups/statements to check for applicable conditions like `TargetFrameworkIdentifier`. Would that work?

Reply:
This is different now I would add a check for `.NETFramework` as a `TargetFrameworkIdentifier` here: https://github.com/onovotny/MSBuildSdkExtras/blob/master/MSBuild.Sdk.Extras/build/Platform.targets#L21 and then add a `platforms\NETFramework.targets` where you can do whatever checks are needed to determine if the WPF stuff should be included. Would that work>